package_manifest.yaml

```diff
@@ -501,6 +501,8 @@ foreman_proxy_plugin_packages:
     rubygem-foreman_probing_core: {}
     rubygem-foreman_remote_execution_core: {}
     rubygem-foreman-tasks-core: {}
+    rubygem-smart_proxy_dynflow_core: {}
```

Comment:
This sorting looks odd

Reply:
I know. I can go back to alphabetical. Things are just weird with this one cause its a stand alone process that can also be a smart proxy plugin (working on sorting this out).